src/views/Bridge.vue

```diff
@@ -299,29 +299,29 @@ export default class Bridge extends Mixins(
   get isInsufficientXorForFee (): boolean {
     if (!this.xorAsset) return true
 
-    const fpBalance = new FPNumber(this.xorAsset.balance, this.xorAsset.decimals)
-    const fpFee = new FPNumber(this.soraNetworkFee, this.xorAsset.decimals)
+    const fpBalance = this.getFPNumberFromCodec(this.xorAsset.balance, this.xorAsset.decimals)
+    const fpFee = this.getFPNumber(this.soraNetworkFee, this.xorAsset.decimals)
```

Comment:
I suggest that `this.soraNetworkFee` stores `CodecString` value, should `this.getFPNumberFromCodec` used here and in all places where we get `this.soraNetworkFee` balance?